### PR TITLE
docs(guides) Fix getting started output filename mismatch

### LINKED ISSUE #2094: Getting started guide: confusion between main.js vs bundle.js
Just working my way through https://webpack.js.org/guides/getting-started/ . 

Paragraph 2 of #modules talks about dist/bundle.js, but I think if I've followed the instructions correctly that should say dist/main.js

The next section uses a config file that specifies dist/bundle.js as the output, but there's no mention of changing dist/index.html to point to it instead of dist/main.js

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Fixes #2094

Linked PRs (thx @montogeek ):

#2106
#2050
#2030

### CODE REVIEW

File: src/content/guides/getting-started.md [hunk 1]
```diff
@@ -204,7 +205,7 @@ Open `index.html` in your browser and, if everything went right, you should see
 
 The [`import`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import) and [`export`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export) statements have been standardized in [ES2015](https://babeljs.io/learn-es2015/). Although they are not supported in most browsers yet, webpack does support them out of the box.
 
-Behind the scenes, webpack actually "transpiles" the code so that older browsers can also run it. If you inspect `dist/bundle.js`, you might be able to see how webpack does this, it's quite ingenious! Besides `import` and `export`, webpack supports various other module syntaxes as well, see [Module API](/api/module-methods) for more information.
+Behind the scenes, webpack actually "transpiles" the code so that older browsers can also run it. If you inspect `dist/main.js`, you might be able to see how webpack does this, it's quite ingenious! Besides `import` and `export`, webpack supports various other module syntaxes as well, see [Module API](/api/module-methods) for more information.
```
Comment (jeremenichelli) [inline src/content/guides/getting-started.md]: I think that transpile is not the right term and will actually confuse not experienced users. It's true that webpack does some small changes on the require statements, but most of all it just adss an overhead to crawl thourgh the modules.

Do you think we could rephrase this?

Comment (EugeneHlushko) [reply src/content/guides/getting-started.md]: the sentence itself is catchy so maybe just change `"transiples"` into `transforms` ?

**Comment by montogeek** [comment]
@EugeneHlushko This PR as been posted several times, every one with a different title, Can you please try to recopile all of them in a single commit and link them there?
https://github.com/webpack/webpack.js.org/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+filename

File: src/content/guides/getting-started.md (hunk 1 above)
Comment (montogeek) [reply src/content/guides/getting-started.md]: Maybe in another PR? We could use this PR to finally settle that filename change.

**Comment by montogeek** [comment]
Yeah, #2032 is another change.

File: src/content/guides/getting-started.md (hunk 1 above)
Comment (EugeneHlushko) [reply src/content/guides/getting-started.md]: okay 👍

**Comment by EugeneHlushko** [comment]
@montogeek ok seems those 3 prs get resolved with this one and the linked issue

**Pull request merged**